scss/_buttons.scss

```diff
@@ -81,7 +81,7 @@
 
   &:active,
   &.selected {
-    text-shadow: 0 1px 0 rgba(0,0,0,0.15);
+    text-shadow: 0 1px 0 rgba(0, 0, 0, 0.15);
```

Comment:
Properties should be ordered background-color, background-image, border-color, text-shadow